packages/docusaurus-plugin-content-blog/src/__tests__/__fixtures__/website/blog/long-text-r.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Long Text
+date: 2021-02-25
+---
+
+Elit irure et ullamco est reprehenderit sint labore incididunt officia voluptate adipisicing occaecat sit. Id aliqua pariatur Lorem nulla dolor ad non nostrud excepteur sint anim officia minim. Aliqua elit est occaecat et ut sit minim. Laboris cupidatat anim ea ipsum esse pariatur. Consequat anim fugiat voluptate laboris elit mollit in dolor laborum.
```

Comment:
This blog post has very simple content, just using strings.

Please use complex markdown, include MDX components like tabs, and make sure that those MDX components are transformed correctly before being sent to the feed.